package.json

```diff
@@ -38,9 +38,7 @@
     "ember-data": "^2.4.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-frost-css-core": "^1.1.3",
-    "ember-frost-icons": "^1.5.0",
-    "ember-frost-theme": "^1.4.0",
+    "ember-frost-core": "0.0.13 - 1",
```

Comment:
Again, this seems like a very odd version to pin to (and not sure why we're pinning the version explicitly. Does it really need EXACTLY that version?